integrations/templates/troubleshooting.md

```diff
@@ -1,11 +1,9 @@
 [% if entry.troubleshooting.list or entry.integration_type == 'collector' or entry.integration_type == 'notification' %]
-## Troubleshooting
-
 [% if entry.integration_type == 'collector' %]
-[% if entry.plugin_name == 'go.d.plugin' %]
+[% if entry.meta.plugin_name == 'go.d.plugin' %]
 ### Debug Mode
```

Comment:
Level 2 heading?

Reply:
yep, nice catch

Reply:
actually no

Reply:
Why?

Reply:
setup has three headings too, let's keep it like this, so Learn doesn't need to juggle hashtags to make stuff

Reply:
Ok. Next question then

> I also removed the ## Troubleshooting
> s it seems we don't do this for other categories, we don't need to state the top heading.

We need. Please add it back.

Reply:
will check what looks best on testing and add/remove accordingly